.bumpversion.cfg

```diff
@@ -1,7 +1,7 @@
 [bumpversion]
 commit = True
 tag = True
-current_version = 0.1
+current_version = 0.8.0
```

Comment:
We don't use this file anymore, can be deleted.

Reply:
Will delete on the 0.8.0 PR.